src/test/java/uk/gov/pay/api/utils/mocks/ConnectorMockClient.java

```diff
@@ -244,6 +248,25 @@ public void respondWithChargeFound(long amount, String gatewayAccountId, String
                         .withBody(chargeResponseBody));
     }
 
+    public void respondWithChargeFoundAndCaptureUrl(long amount, String gatewayAccountId, String chargeId, PaymentState state, String returnUrl,
```

Comment:
Rather than adding a whole new method, we could instead modify the existing `respondWithChargeFound` so that if the status is `AWAITING_CAPTURE_REQUEST`, it adds a link with the capture URL. Then the test would just need to call `respondWithChargeFound` and pass in the appropriate status.